src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -833,6 +833,7 @@ def load_arguments(self, _):
         c.argument('max_surge', validator=validate_max_surge)
         c.argument('drain_timeout', type=int)
         c.argument('node_soak_duration', type=int)
+        c.argument("undrainable_node_behavior", type=str, default='Schedule')
```

Comment:
```suggestion
        c.argument("undrainable_node_behavior", default='Schedule')
```
the default type is str, may omit it